spring-cloud-gcp-starters/spring-cloud-gcp-starter-vision/src/main/resources/META-INF/spring.provides

```diff
@@ -1 +1 @@
-provides: spring-cloud-gcp-starter
+provides: spring-cloud-gcp-starter, spring-cloud-gcp-vision
```

Comment:
According latest Spring Boot such `/META-INF/spring.provides` files can be removed: https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.1-Release-Notes#removal-of-springprovides-files

We can do that in the separate PR though, for the whole project.

Reply:
Noted. Sounds good.